make_md.py

```diff
@@ -49,7 +54,14 @@ def convert_from_bib(myline):
         print(myentry)
         print("We are in trouble ! ")
     if "eprint" in myentry_dict:
-        return "["+myentry_dict["title"]+"](https://arxiv.org/abs/"+myentry_dict["eprint"]+")"
+        paper=""
+        if "doi" in myentry_dict:
+            paper=", [[DOI](https://doi.org/{0})]".format( myentry_dict["doi"] )
```

Comment:
```suggestion
            paper=" [[DOI](https://doi.org/{0})]".format( myentry_dict["doi"] )
```

Reply:
Probably no need for the comma, given that the square brackets clearly delineate the change.